civihr_default_theme/scss/components/_table.scss

```diff
@@ -0,0 +1,11 @@
+.chr_table--clean {
```

Comment:
You sure it needs to be a custom component? Why can't it a variation of bootstrap's .table?